databook/physiology/ephys/np-ultra/npultra-psychedelics-stimuli.md

```diff
@@ -0,0 +1,14 @@
+# NP Ultra & Psychedelics Stimuli and Optotagging protocol
+
+## Spontaneous Activity
+Each session includes at least three epochs of spontaneous activity. During this epoch the monitor is held at mean luminance gray and there is no patterned stimulus presented. This provides a valuable time that can be used as a baseline comparison for visually evoked activity.
+
+## Receptive Field Mapping with Gabor patches
+The gabor patch stimulus consists of sinuisoidal gratings that are spatially restricted. Key parameters include orientation and direction (in degrees), temporal frequency (in Hz), spatial frequency (in cycles/deg), contrast (in percent), patch size (in degrees), and patch location (in degreees). Each trial is presented for 0.25 seconds.
```

Comment:
if some of these parameters are fixed, it'd be useful to say what they are? Eg patch size? Maybe contrast, SF, TF? How many orientations are used? How many locations do you use?